integration_tests/macros/unit_tests/test_get_test_min_and_max_buckets.sql

```diff
@@ -0,0 +1,276 @@
+{% macro test_get_test_buckets_min_and_max() %}
+
+    {%- set monitors_runs_schema = [('full_table_name', 'string'), ('metric_properties', 'string'), ('last_bucket_end', 'timestamp'), ('first_bucket_end', 'timestamp')] %}
```

Comment:
Missing `metric_name` column (Not sure if on purpose)

Reply:
On purpose